src/components/Display.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import '../index.css';
+
+export default class Display extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      result: '0',
+    };
+  }
+
+  render() {
+    const { result } = this.state;
+    return <div>{result}</div>;
+  }
```

Comment:
**Display component**
- it should accept the result of the calculation as an incoming **prop** not as a **state**  see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/incomingProps.png) of requirement
- the result prop should be a String with a default value of `"0"` (string)  [should be String screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)   [default props screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)

Reply:
Thank you for pointing it out! It has been fixed :)